src/routes/api/collabs/[id]/picks/[pick_id]/index.ts

```diff
@@ -199,9 +199,7 @@ export const del: RequestHandler = async ({ request, params }) => {
 	const decodedUser = Jwt.decode(cookies['user_id']);
 	const userId = decodedUser['user_id'] as string;
 
-	const reason = await request.json().then(function (data) {
-		return data.Reason;
-	});
+	const reason = (await request.json()).reason
```

Comment:
Think this would be cleaner if you put the request.json into a const itself and then call .reason on that const to create another const.

Just looks cleaner imo and should aid to remove the () wrapping

Reply:
Yea i think it would be more cleaner also it would be better in the future if we put more parameters.